tests/kubernetes.rs

```diff
@@ -0,0 +1,427 @@
+// NOTE: Tests assume that Kubernetes is accessable and localy available image of vector
```

Comment:
I think it may be better to move this to `src/sinks/kube/test.rs` then bring it in as a `mod test` under `#[cfg(test)`. The reason for this is every `tests/*.rs` we have will have to link against a whole copy of vector and this makes tests much slower. So we should be doing as much of the tests within the main binary under the `cfg(test)` attribute.

Reply:
Done, and I have also moved source to be under the same mod.rs .